pkg/rid/cockroach/store.go

```diff
@@ -84,7 +84,7 @@ func recoverRollbackRepanic(ctx context.Context, tx *sql.Tx) {
 // NewStore returns a Store instance connected to a cockroach instance via db.
 func NewStore(db *cockroach.DB, logger *zap.Logger) *Store {
 	return &Store{
-		ISA:          &ISAStore{db, DefaultClock, logger},
```

Comment:
Why do we want to drop the clock specification here?  This seems disconnected from the rest of the PR

Reply:
I think once we introduce GC we don't need a clock on the store at all, which should give a cleaner repo API. Hopefully dropping it now will discourage it's use until we do that.